examples/ConfigOnDoubleReset/ConfigOnDoubleReset.ino

```diff
@@ -0,0 +1,104 @@
+/*
```

Comment:
Yes @tablatronix , but I'd be surprised if the commercial device resets though. 

A led that flashes faster while the button is pressed, eventually becoming solid is what the [Google Chromecast](https://www.google.com/intl/en_au/chromecast/tv/chromecast/) does. I like it and copied the idea in [this sketch](https://github.com/kentaylor/esp8266-weather-station/blob/master/examples/WeatherDHT22ThreeScreensWiFiManager/WeatherDHT22ThreeScreensWiFiManager.ino) where a progress bar on a screen is used to launch the configuration portal. In the sketch it uses this [button library](https://github.com/kentaylor/OneButton) that can detect press, double press, long press and long press followed by still pressed. The button becomes multi purpose then.

I was thinking of adding it with flashing led instead of progress bar as a WiFiManager example that behaves like a Chromecast but it's a bit of effort.